tests/fixtures/src/jvmMain/kotlin/io/spine/testing/logging/TapConsole.kt

```diff
@@ -24,7 +24,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.logging
+package io.spine.testing.logging
```

Comment:
I would suggest to have something like `...testutil` module for `TapConsole`, similarly to what we have in Spine for other testing-specific routines. 

"A test fixture" — it's where it resides now — is something different to me. More like a collection of routines constructing a suitable context for tests to make sense.

@yevhenii-nadtochii @alexander-yevsyukov WDYT?

Reply:
`logging-testutil` looks like a reasonable thing. We did have some tests for logging in `core-java`, if memory serves. We also have JUL-based testing utilities in `testlib`. They should be deprecated in favor of newer log testing utils.

Reply:
Moved `TapConsole` to `:testutil`. 

Until this module is used outside `logging`, it can be just `testutil` (without `logging` prefix).